doxygen/Doxyfile

```diff
@@ -53,23 +63,41 @@ PROJECT_BRIEF          =
 
 PROJECT_LOGO           =
 
+# With the PROJECT_ICON tag one can specify an icon that is included in the tabs
+# when the HTML document is shown. Doxygen will copy the logo to the output
+# directory.
+
+PROJECT_ICON           =
```

Comment:
Should we add our simplified Ice icon here?

Reply:
#1868